src/database/mongodb_client.py

```
@@ -192,4 +201,40 @@ def close_connection(self):
+    def get_active_keywords(self, limit: int = 100) -> List[str]:
+        """Get active keywords for data collection"""
+        try:
+            doc = self.db.settings.find_one({'_id': 'keywords'})
+            if doc and 'active' in doc:
+                return doc['active'][:limit]
+            return []
+        except Exception as e:
+            logging.error(f"Error getting active keywords: {e}")
+            return []
+
+    def set_active_keywords(self, keywords: List[str], user: Optional[str] = None) -> None:
```

Ruff (0.14.5)

Unused method argument: `user`

(ARG002)